homeassistant/components/remote/harmony.py

```diff
@@ -25,12 +25,13 @@
 
 DEFAULT_PORT = 5222
 DEVICES = []
+CONFIGS = []
```

Comment:
The way the override works, I read values from the config argument. If the config includes a port/host, I pull all values from it and set up the hub. However, if the config does not include the port/host value (and is therefore "incomplete") I don't create the hub device, but I store its configuration in CONFIGS variable. Then later, when autodiscover happens later, I look through these incomplete configs, find one that matches the name of the host that has been found, and merge the found host/ip with the rest of the config then at that point create the device. So in the end I do need this var outside of `setup_platform`.

I'm new to HA component code so let me know if there's a better way.